docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,191 @@
+| 200
+
+// ==============================================================
+
+| http.version
+| 1.1
+
+// ==============================================================
+
+| source.address
+| 10.42.42.42
+
+// ==============================================================
+
+| source.ip
+| 10.42.42.42
+
+// ==============================================================
+
+| url.original
+| `/content`
+
+// ==============================================================
+
+| user_agent.device.name
+| Other
+
+// ==============================================================
+
+| user_agent.name
+| Chrome
+
+// ==============================================================
+
+| user_agent.original
+| `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36`
+
+// ==============================================================
+
+| user_agent.os.full
+| Mac OS X 10.15.4
+
+// ==============================================================
+
+| user_agent.os_name
+| Mac OS X
+
+// ==============================================================
+
+| user_agent.os.version
+| 10.15.4
+
+// ==============================================================
+
+| user_agent.os.version
+| 83.0.4103.106
+
+|=====
+
+Some items to note on the mapping example:
+
+* As many ECS fields as possible are mapped from the original event
+* Several `event` fields are populated based on ECS categorization
+* The raw log is preserved in the `event.original` field
+* For the simplicity of this example some field sets such as `agent.*`
+  and `host.*` have been omitted.
+
+
+[float]
+==== Simplified Search
+
+With ECS defining a normalized schema across all of your data sources, querying
+against those sources is simplified. Consider searching for a particular source IP
+address prior to adopting ECS. All the various data sources and their field mappings
+would need to be considered in your query:
+
+[source,sh]
+-----------
+src:10.42.42.42 OR client_ip:10.42.42.42 OR apache.access.remote_ip:10.42.42.42 OR
+context.user.ip:10.42.42.42 OR src_ip:10.42.42.42
+-----------
+
+With all sources mapped to ECS, the query becomes much simpler:
+
+[source,sh]
+-----------
+source.ip:10.42.42.42
+-----------
+
+Not only does this simplify writing queries, but any saved or shared queries between
+users become much more obvious. Any unfamiliar fields that align with ECS can also
+be reviewed in the <<ecs-field-reference>> to gain better understanding.
+
+[float]
+==== Unified Visualizations
+
+With normalized data from different data sources, building insightful visualizations
+across sources is also greatly simplified. From a single, centralized dashboard, events
+from web servers, IDS/IPS devices, and firewalls can be aggregated, visualized, provides
+drill-down, and then pivoted from into deeper investigations. Centralized monitoring of
+diverse data sources has been greatly simplified normalizing to ECS.
```

Comment:
```suggestion
from web servers, IDS/IPS devices, and firewalls can be aggregated and visualized, and enhanced with
drill-downs and pivots for delving into deeper investigations. Centralized monitoring of
diverse sources is straightforward with normalized ECS data.
```
Not sure this edit keeps your original intent.

Reply:
It does - thanks for the suggestions!